roles/tas_single_node/defaults/main.yml

```diff
@@ -19,7 +19,14 @@ tas_single_node_skip_os_install: false
 
 tas_single_node_rekor_templates:
   - manifests/rekor/redis-server.yaml
-  - manifests/rekor/rekor-server.yaml
+  - manifests/rekor/rekor-server.j2
+
+tas_single_node_rekor_redis:
+  database_deploy: true
+  redis:
+    address: "rekor-redis"
```

Comment:
I'm wondering why this isn't `rekor-redis-pod`. I understand that this is the value that was used before and you just extracted it to this variable, but I'm struggling to understand why in other places we use `servicename-pod` syntax and here we just use `servicename`. Are both possible? Could you please look into this so that we understand? Thanks!

Reply:
you're right! I'm wondering how it was "working" 
![image](https://github.com/user-attachments/assets/913db175-8ee1-4051-9a17-5e1e042229b0)

Reply:
Maybe it didn't, but rekor can run without redis? Maybe it only uses it for caching, but can actually work (slower) without it?

Reply:
yes, it's used for caching, it wasn't detected with the actual tests, since we don't test all rekor functionalities (and performance that depends on caching)